x-pack/plugins/observability/public/components/app/section/index.tsx

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { EuiAccordion, EuiLink, EuiPanel, EuiSpacer, EuiText, EuiTitle } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import styled from 'styled-components';
+import { ErrorPanel } from './error_panel';
+
+interface Props {
+  title: string;
+  subtitle: string;
+  hasError: boolean;
+  children: React.ReactNode;
+  minHeight?: number;
+  appLink?: string;
+}
+const StyledEuiAccordion = styled(EuiAccordion)`
+  .euiAccordion__triggerWrapper {
+    border-bottom: ${(props) => props.theme.eui.euiBorderThin};
+  }
+  .euiAccordion__button {
+    margin-bottom: 16px;
```

Comment:
```suggestion
    margin-bottom: 8px;
```
Looks like we don't need more than 8px now that the border is gone.